src/devtools/makefile_base_posix.mak

```diff
@@ -421,7 +421,7 @@ else
 endif
 
 ifneq "$(origin VALVE_NO_AUTO_P4)" "undefined"
-	P4_EDIT_START = chmod -R +w
+	P4_EDIT_START = chmod -fR +w
```

Comment:
Why -f?

Reply:
I got rid of this in exchange for another change. This is to silence chmod errors when a file doesn't exist, which the new commit fixes by moving the invocations to *after* when they're built. But since I don't know what P4_EDIT_START does when perforce stuff is enabled, I'm not sure if this is a good idea... It may just be best to use this instead

Reply:
I've decided to go back to using `chmod -f`, since all that does is silence errors. Since the errors don't happen on rebuilds, only fresh builds, i'm going to assume it's just an artifact of how the perforce stuff works, and not anything to worry about.